llrt_core/src/modules/events/mod.rs

```diff
@@ -144,6 +160,22 @@ where
         Ok(proto)
     }
 
+    fn add_event_target_prototype(ctx: &Ctx<'js>) -> Result<Object<'js>> {
+        let proto = Class::<Self>::prototype(ctx.clone())
+            .or_throw_msg(ctx, "Prototype for EventTarget not found")?;
+
+        let on = Function::new(ctx.clone(), Self::evt_add_event_listener)?;
```

Comment:
How is this different from regular event emitter method? Can't we just call that instead?

Reply:
`EventTarget.dispatchEvent()` and `EventEmitter.emit()` have the same functionality but different arguments.

- EventTarget: dispatchEvent() method
https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/dispatchEvent
- EventEmitter.emit()
https://nodejs.org/api/events.html#emitteremiteventname-args

Also, `EventTarget.addEventListener()` has an `once` option, which at least allows the functionality equivalent to `EventEmitter.on()` and `EventEmitter.once()` to be controlled by this option.

- EventTarget: addEventListener() method
https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener

Because of this difference, we have prepared an implementation that can add the necessary prototype methods to `EventTarget`.